tinywallet/tinywallet/app.py

```diff
@@ -117,12 +100,12 @@ def __init__(self, qApp):
         self.loadSettings()
 
         dcrdataDB = database.KeyValueDatabase(
-            os.path.join(self.netDirectory(), "dcr.db")
+            os.path.join(self.netDirectory(), "wallet.db")
```

Comment:
Forgot to revert?

Reply:
What the heck...? I'm pretty sure I had reverted it. Thanks for the heads up.